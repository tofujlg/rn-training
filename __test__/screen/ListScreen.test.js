import React from 'react';
import  { create } from 'react-test-renderer';
import ListScreen from '../../screen/ListScreen';
ListScreen


const tree = create(<ListScreen/>)

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});