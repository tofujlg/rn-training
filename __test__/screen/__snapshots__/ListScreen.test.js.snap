// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View>
  <Text>
    ListScreen
  </Text>
</View>
`;
